theme/static/style/style.css

```diff
@@ -454,6 +454,75 @@ pre code {
     }
 }
 
+@media screen and (max-width: 991px){
+    /* For tablets: */
+    /*Screenshots section.*/
+    .arrow-left, .arrow-right, .img-index {
+        display: none;
+        width: 0;
+        height: 0;
+    }
+
+    .img-wrapper {
+        width: 100%;
+    }
+
+    .img {
+        bottom: 3rem;
+    }
+}
+
+@media screen and (max-width: 767px) {
```

Comment:
We already have the same selector block, so no need to repeat it twice. Just move the contents under the existing block.